frontend/src/lib/components/feedback/Feedback.svelte

```diff
@@ -76,6 +76,19 @@
     rating = undefined;
     description = '';
     status = 'default';
+    if (zeroInput) zeroInput.checked = true;
```

Comment:
We use this to reset the star gauge, which is not bidirectionally bound to `rating`.

Reply:
And that's bc it's a DaisyUI component.